stellarphot/core.py

```diff
@@ -963,6 +963,65 @@ def from_vizier(
 
         return cat
 
+    def passband_columns(self, passbands=None):
+        """
+        Return an `astropy.table.Table` with passbands as column names instead
+        of the default format, which has a single column for passbands.
+
+        Parameters
+        ----------
+        passbands : list, optional
+            List of passbands to include in the output. If not provided, all
+            passbands in the catalog will be included.
+
+        Returns
+        -------
+        `astropy.table.Table`
+            Table of catalog information with passbands as column names. See Notes below
+            for important details about column names.
+
+        Notes
+        -----
+
+        The column names in the output will be the passband names with ``mag_`` as a
+        prefix. An error column for each passband will be generated, with the prefix
+        ``mag_error_``. If the catalog already has columns with these names, they will
+        be overwritten. The input catalog will not be changed.
+        """
+        catalog_passbands = set(self["passband"])
```

Comment:
Finally, a use for `set` that isn't students trying to de-duplicate items in a list by looking up answer in Stack Overflow.

Reply:
🤣